OrbitGl/TimeGraph.cpp

```diff
@@ -621,30 +621,37 @@ std::string GetTimeString(const TimerInfo& timer_a, const TimerInfo& timer_b) {
   return colors[index % kNumColors];
 }
 
-void DrawIteratorBox(GlCanvas* canvas, Vec2 pos, Vec2 size, const Color& color,
+}  // namespace
+
+void TimeGraph::DrawIteratorBox(GlCanvas* canvas, Vec2 pos, Vec2 size, const Color& color,
                      const std::string& label, const std::string& time, float text_y) {
   Box box(pos, size, GlCanvas::kZValueOverlay);
   canvas->GetBatcher()->AddBox(box, color);
 
   std::string text = absl::StrFormat("%s: %s", label, time);
 
   constexpr const float kAdditionalSpaceForLine = 10.f;
-  constexpr const float kLeftOffset = 5.f;
+  const Color kWhite(255, 255, 255, 255);
+  const Color kBlack(0, 0, 0, 255);
 
   float max_size = size[0];
-  canvas->GetTextRenderer().AddTextTrailingCharsPrioritized(
-      text.c_str(), pos[0] + kLeftOffset, text_y + kAdditionalSpaceForLine, GlCanvas::kZValueText,
-      Color(255, 255, 255, 255), time.length(), GParams.font_size, max_size);
+  float text_width = canvas->GetTextRenderer().AddTextTrailingCharsPrioritized(
+      text.c_str(), pos[0], text_y + kAdditionalSpaceForLine, GlCanvas::kZValueText,
+      kBlack, time.length(), GParams.font_size, max_size);
 
   constexpr const float kOffsetBelowText = kAdditionalSpaceForLine / 2.f;
+  Vec2 white_box_size(text_width, GetTextBoxHeight());
+  Vec2 white_box_position(pos[0], text_y + kOffsetBelowText);
+
+  Box white_box(white_box_position, white_box_size, GlCanvas::kZValueOverlay);
+  canvas->GetBatcher()->AddBox(white_box, kWhite);
```

Comment:
I would prefer that the white line for the measuring is in the middle of the box (y-coordinate)
![Screen Shot 2020-10-01 at 12 09 50](https://user-images.githubusercontent.com/8610429/94796675-63affb80-03df-11eb-90bd-96e1b633095d.png)

Reply:
Agree, done.